common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VroPackageStore.java

```diff
@@ -122,7 +171,11 @@ public Package exportPackage(Package vraPackage, boolean dryrun) {
     @Override
 	public Package exportPackage(Package vraPackage, VroPackageDescriptor vroPackageDescriptor, boolean dryrun) {
 		logger.info(String.format(PackageStore.PACKAGE_EXPORT, vraPackage));
-		Package pkg = restClient.exportPackage(vraPackage, dryrun);
+
+		boolean exportConfigAttributeValues = false;
```

Comment:
These both should be true. The primary purpose of this backup is to export the values in the configuration elements.

Reply:
@ivo-kotev the same comment like above